lib/cluster.js

```diff
@@ -261,7 +261,7 @@ function masterInit() {
     assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
            'Bad cluster.schedulingPolicy: ' + schedulingPolicy);
 
-    var hasDebugArg = process.execArgv.some(function(argv) {
+    var hasDebugArg = process.execArgv.some(function hasDebugTest(argv) {
```

Comment:
Ref to CTC meeting, please? I saw another PR this morning where named functions are being changed to anon es6.

Btw, this is third time I've asked for a reference to the decision that motivates this.